### ISSUE TITLE
Open other pane/window

### ISSUE THREAD

**Issue by zadca123** [vifm/vifm#983]

**Comment by yusufaktepe** [comment]
From the default [vifmrc](https://github.com/vifm/vifm/blob/master/data/vifmrc#L537-L538)
```vim
" Open directory under cursor in the other pane
nnoremap <a-o> :sync %c<cr>
```

**Comment by xaizek** [comment]
Depends on what you mean.  Because Vifm runs external applications to open files, you can't really open files inside Vifm, only preview.  But for directories <kbd>Alt+o</kbd> keybinding posted above might be what you're looking for.

**Comment by zadca123** [comment]
Yes, it was I was looking for. In my old config it was `a i` but `Alt+o` works too.  Thank you!
